pkgs/tools/security/fwbuilder/default.nix

```diff
@@ -1,4 +1,18 @@
-{ stdenv, lib, fetchFromGitHub, cmake, qtbase, wrapQtAppsHook }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qtbase
+, wrapQtAppsHook
+, waylandSupport ? true
```

Comment:
```suggestion
```

Reply:
removing this:

```
$ head ./pkgs/tools/security/fwbuilder/default.nix
{ lib
, stdenv
, fetchFromGitHub
, cmake
, qtbase
, wrapQtAppsHook
, wayland
, wayland-protocols
, qtwayland
}:
```

Makes the build fail:

```
$ nix-build $NIXPKGS -A fwbuilder
error: undefined variable 'waylandSupport' at /data/nixpkgs/pkgs/tools/security/fwbuilder/default.nix:28:31
```

And removing the optional flag also fails:

```
$ head ./pkgs/tools/security/fwbuilder/default.nix
{ lib
, stdenv
, fetchFromGitHub
, cmake
, qtbase
, wrapQtAppsHook
, waylandSupport
, wayland
, wayland-protocols
, qtwayland

[elatov@nixos:/data/nixpkgs]$ nix-build $NIXPKGS -A fwbuilder
error: Function called without required argument "waylandSupport" at /data/nixpkgs/pkgs/tools/security/fwbuilder/default.nix
```

Reply:
you, of course, need to remove the uses of it